api/user.py

```diff
@@ -1,7 +1,14 @@
 from flask_login import UserMixin
 
+from enum import IntEnum
+
 import faf.db as db
 
+class UserGroup(IntEnum):
+    NONE = 0,
+    MODERATOR = 1,
+    ADMIN = 2
```

Comment:
Also, the 0/1/2 levels were already there before. So would need changing the schema. That can be done in a separate PR.